packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2467,18 +2450,36 @@ export class LitNodeClientNodeJs
       params.expiration ||
       new Date(Date.now() + 24 * 60 * 60 * 1000).toISOString();
 
-    // Try to get it from local storage, if not generates one~
-    const sessionKey = params.sessionKey ?? this.getSessionKey();
-    const sessionKeyUri = LIT_SESSION_KEY_URI + sessionKey.publicKey;
+    let sessionKeyUri: string;
+
+    // This allow the user to provide a sessionKeyUri directly without using the session key pair
```

Comment:
But we still need the sessionKey is it created from the sessionKeyUri?

Reply:
actually this shouldn't be here, it was part of the code while developing. It's irrelevant now as we need to always pass or generate session key